orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/ManifestStrategyType.java

```diff
@@ -0,0 +1,25 @@
+package com.netflix.spinnaker.orca.clouddriver.tasks.manifest;
+
+public enum ManifestStrategyType {
+  RED_BLACK("redblack"),
+  HIGHLANDER("highlander"),
+  NONE("none");
+
+  String key;
+
+  ManifestStrategyType(String key) {
+    this.key = key;
+  }
+
+  public static ManifestStrategyType fromKey(String key) {
```

Comment:
Yeah I had figured it's better to have the logic to convert from string to enum within the `ManifestStrategyType` class itself rather than using `valueOf` within a try/catch everywhere we're trying to do this, just so the default is a part of the class and not up to the consumer

Reply:
Sounds good, yeah not really sure my suggestion would make things any cleaner.